mirror-service/src/test/java/org/dependencytrack/vulnmirror/KafkaStreamsTopologyIT.java

```diff
@@ -288,34 +292,36 @@ record -> {
 
     @QuarkusIntegrationTest
     @TestProfile(OsvMirrorIT.TestProfile.class)
+    @ConnectWireMock
     static class OsvMirrorIT {
 
         public static class TestProfile implements QuarkusTestProfile {
+
+            @Override
+            public Map<String, String> getConfigOverrides() {
+                return Map.of("mirror.datasource.osv.base-url", "http://localhost:${quarkus.wiremock.devservices.port}");
+            }
+
             @Override
             public List<TestResourceEntry> testResources() {
                 return List.of(
-                        new TestResourceEntry(KafkaCompanionResource.class),
-                        new TestResourceEntry(
-                                WireMockTestResource.class,
-                                Map.of("serverUrlProperty", "mirror.datasource.osv.base-url")
-                        ));
+                        new TestResourceEntry(KafkaCompanionResource.class));
             }
         }
 
         @InjectKafkaCompanion
         KafkaCompanion kafkaCompanion;
 
-        @InjectWireMock
-        WireMockServer wireMock;
+        WireMock wireMock;
 
         @Test
         void test() throws Exception {
             // Simulate the first page of CVEs, containing 2 CVEs.
-            wireMock.stubFor(get(urlPathMatching("/.*"))
+            wireMock.register(get(urlPathMatching("/.*"))
                     .willReturn(aResponse()
                             .withStatus(200)
                             .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON)
-                            .withResponseBody(Body.ofBinaryOrText(resourceToByteArray("/datasource/osv/maven.zip"), new ContentTypeHeader(MediaType.APPLICATION_OCTET_STREAM)))));
+                            .withResponseBody(Body.ofBinaryOrText(resourceToByteArray("/datasource/osv/maven.zip"), new ContentTypeHeader(MediaType.APPLICATION_JSON)))));
```

Comment:
I think the previous value was correct here since the response body is a ZIP archive (binary), not JSON.

Reply:
It seems Quarkus wiremock doesn't support zip compression header type. 
When I'm using `APPLICATION_OCTET_STREAM` it throws `EOFException`. 
Similar for Epss test case, the error throws `ZipException: Not in GZIP format`